core/storage/base_model/gae_models.py

```diff
@@ -889,17 +889,19 @@ def _require_not_marked_deleted(self) -> None:
         if self.deleted:
             raise Exception('This model instance has been deleted.')
 
-    # TODO(#13523): Change 'snapshot' to domain object/TypedDict to
-    # remove Any from type-annotation below.
+    # Here we use type Any because the method 'compute_snapshot' defined in
+    # subclasses of VersionedModel can return different Dict/TypedDict types.
+    # So, to allow every Dict/TypedDict type we used any here.
     def compute_snapshot(self) -> Dict[str, Any]:
         """Generates a snapshot (dict) from the model property values."""
         return self.to_dict(exclude=['created_on', 'last_updated'])
 
-    # TODO(#13523): Change 'snapshot_dict' to domain object/Typed Dict to
-    # remove Any from type-annotation below.
+    # Here we use type Any because the method '_reconstitute' defined in
```

Comment:
Same thing (and in fact that's in line with the TODO comment that was here previously).

Reply:
Since this method is defined in base class and redefined in other subclasses -- the above reasoning is also applied here.

I'm not sure why previously we have This todo here because BasseModel does not have any domain object specifically.